sdk/include/opentelemetry/sdk/resource/resource.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include "opentelemetry/sdk/common/attribute_utils.h"
+#include "opentelemetry/sdk/resource/resource_detector.h"
+#include "opentelemetry/sdk/version/version.h"
+#include "opentelemetry/version.h"
+
+#include <cstdlib>
```

Comment:
Is this include necessasry?

Reply:
No, we actually need it in `resource.cpp`. Will move it there. Thanks for noticing it.

Reply:
This is done now.